test/readme.md

```diff
@@ -12,6 +12,10 @@ For the e2e, production, and development tests the `createNext` utility should b
 
 All new test suites should be written in TypeScript either `.ts` (or `.tsx` for unit tests). This will help ensure we catch smaller issues in tests that could cause flakey or incorrect tests.
 
+You can set-up a new test using `yarn new-test` which will start from a template related to the test type.
```

Comment:
Lets move this to the top? I think we might be burying the lede

Reply:
Updated in https://github.com/vercel/next.js/pull/33001/commits/e338dbf409b32f79d76d75d7dd5a93159d8d251c